### PR TITLE
Cartsection

### PR DESCRIPTION
**Pull request by mdrdavid**
Updated more features

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,18 +1,37 @@
...
+import { Routes, Route} from 'react-router-dom';
+import CartSection from './pages/cartsection/CartSection';
 import Home from './pages/home/Home';
-// import Login from '../login/Login';
+import Login from './pages/login/Login';
+import SingleProduct from './pages/singleproduct/SingleProduct';
+import Register from './pages/register/Register';
+import Shipping from './pages/shipping/Shipping';
+import PlaceOrder from './pages/placeorder/PlaceOrder'
+import ProfilePage from "./pages/profile/ProfilePage"
```
Comment (kcharles52) [inline src/App.js:11]: Consistence is needed, use of double quotes

File: src/components/homeComponents/shopsection/ShopSection.js [hunk 2]
```diff
@@ -2,48 +2,48 @@ import React from 'react'
 import Rating from '../Rating'
 import Pagination from '../Pagination'
 import { Link } from 'react-router-dom'
+import Products from '../../data/Products'
+import "./shopsection.css"
 
  const  ShopSection= ()=> {
-     const products =["sneakers", "jeans", "sandles"]
+console.log("products",Products.image)
```
Comment (kcharles52) [inline src/components/homeComponents/shopsection/ShopSection.js]: These are meant for debugging and should be removed when done

File: src/components/homeComponents/shopsection/ShopSection.js [hunk 3]
```diff
@@ -2,48 +2,48 @@ import React from 'react'
...
-                                            <img src={product.image} alt={product.name}/>
+                                            <img src={Products.image} alt={Products.name}/>
                                             </div>
                                     </Link>
                                     <div className='shoptext'>
                                         <p>
-                                            {/* <Link to={`/products/${product._id}`}
-                                            // {...product.name}
+                                            {/* <Link to={`/Productss/${Products._id}`}
+                                            // {...Products.name}
```
Comment (kcharles52) [inline src/components/homeComponents/shopsection/ShopSection.js]: Its good practice to remove commented code when done

File: src/components/homeComponents/shopsection/shopsection.css [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    height: 100px;
+}
+.border-product{
+    border: 1px solid black;
+    width: 200px;
+    height: 100px;
+}
+.shoptext{
+    font-size: 18px;
+}
```
Comment (kcharles52) [inline src/components/homeComponents/shopsection/shopsection.css:40]: In programming we try to leave a blank line at the end of our code files

**Review by kcharles52 — CHANGES_REQUESTED** [review]
Great job @mdrdavid, This is great progress
Please work on the comments added and where they apply to other files. Also try to create pull request with few files changed, you have over 50, this makes reviewing difficult.

**Comment by mdrdavid** [comment]
I will effect the changes
Thanks

**Pull request merged**